include/class.forms.php

```diff
@@ -1379,6 +1380,29 @@ function getConfigurationOptions() {
         );
     }
 
+    function validateEntry($value) {
+        parent::validateEntry($value);
+        $config = $this->getConfiguration();
```

Comment:
```php 
if (!$value) return;
```

I would say return early of value is empty - why get configuration etc only to break when `$value` is empty in line 1395

Reply:
Done 👍